org.eclipse.jdt.core/model/org/eclipse/jdt/core/IType.java

```diff
@@ -943,6 +943,16 @@ IType createType(String contents, IJavaElement sibling, boolean force, IProgress
 	 */
 	boolean isSealed() throws JavaModelException;
 
+	/**
+	 * Returns whether this type is unnamed.
+	 *
+	 * @return true if this type is unnamed and false otherwise
+	 * @throws JavaModelException if this element does not exist or if an
+	 *		exception occurs while accessing its corresponding resource.
+	 * @since 3.37
+	 */
+	boolean isUnnamed() throws JavaModelException;
```

Comment:
I know it's still a preview, but would be good if we follow the terms from the spec. I.e., isUnnamed -> isImplicitType() or something like that?

Reply:
> I know it's still a preview, but would be good if we follow the terms from the spec. I.e., isUnnamed -> isImplicitType() or something like that?
Also, the spec does not say anything about preserving this attribute in .class. So, not sure tagging them AccUnnamed is right.

Reply:
Sure, I think it's a better idea to use the new name for the API. I'll update the PR accordingly.